src/noit_message_decoder.c

```diff
@@ -749,6 +765,18 @@ noit_metric_tagset_builder_add_one(noit_metric_tagset_builder_t *builder,
   return mtev_true;
 }
 
+mtev_boolean
+noit_metric_tagset_builder_add_one(noit_metric_tagset_builder_t *builder, const char *tagstr,
+                                    size_t tagstr_len) {
+  return add_tag_to_tagset_builder(builder, tagstr, tagstr_len, NOIT_TAG_MAX_PAIR_LEN);
+}
+
+mtev_boolean
+noit_metric_tagset_builder_add_one_name(noit_metric_tagset_builder_t *builder,
```

Comment:
```suggestion
noit_metric_tagset_builder_add_one_implicit(noit_metric_tagset_builder_t *builder,
```

Reply:
since we only intend to use this for names (all other implicit tags are short enough to be supported by the existing endpoint), I thought it would be safer to explicitly call this `name` so that no one tries to use it for something else.
do we anticipate new implicit tags that may use this in the future?

Reply:
No, ``implicit``, both Brian and I made the point it was to be ``implicit`` because we don't want to add anything if there are further needs

For the rationale, please re-read the original discussions on this matter